bin/load-data

```diff
@@ -4,13 +4,17 @@
 # However, sometimes Solr is still loading and we need to wait until it's done
 # befoe we can begin.
 
-while [[ "$STATUS" != "302" ]]; do
-  echo waiting for setup to complete or equal 302.
+STATUS=$(curl -s -o /dev/null -w "%{http_code}" http://solr:8983/solr/admin/cores)
+
+while [[ "$STATUS" == "200" ]]; do
```

Comment:
```suggestion
while [[ "$STATUS" != "200" ]]; do
```
It doesn't feel right to have a while loop equals 200 and then echo that we are waiting for it to equal 200. Should it be Does NOT equal?

Reply:
@sensei100 thanks... Yeah I think I changed this by mistake.

Reply:
yeah i didn't mean to update that from 302.